packages/react-native-benchmarking-library/src/Benchmark.tsx

```diff
@@ -43,16 +43,20 @@ export const Benchmark: React.FC<BenchmarkProps> = ({
   }
 
   if (running) {
-    return <Text>Running benchmark, please wait</Text>;
+    return (
+      <View style={{ justifyContent: "center", alignItems: "center" }}>
+        <Text>Running benchmark, please wait</Text>
+      </View>
+    );
   }
 
   if (benchmarkRunTime) {
     const completedLabel = `${name}Completed`;
     return (
-      <>
-        <Text testID={completedLabel}>{benchmarkRunTime}</Text>
+      <View style={{justifyContent: 'center', alignItems: 'center'}}>
+        <Text style={{fontWeight: 'bold'}} testID={completedLabel} accessibilityValue={{text: `${benchmarkRunTime}`}}>Benchmark took: {benchmarkRunTime}ms</Text>
```

Comment:
I think this will break our XPath for parsing the benchmark result, since we take the value tagged with provided testId. We would need to either update the XPath in the benchmarking-scripts package or update the parsing of the value in the benchmark-parser package.

Reply:
Ah true, since we copied the library files to benchmarking app I guess we can close the PR